go-app.js

```diff
@@ -203,22 +213,6 @@ go.app = function() {
       });
 
     });
-    self.states.add('No_SA_ID_Month_Entry', function(name) {
-      var question = $("Please enter the month the patient was born. For example: 12");
-      var error = $('');
-      return new FreeText(name, {
-        question: question,
-        check: function(content) {
-          if (true) {
-            return null; // vumi expects null or undefined if check passes
-          } else {
-            return error;
-          }
-        },
-        next: 'No_SA_ID_Gender_Entry'
```

Comment:
I fixed this above, it now goes to the day after asking the month. The Day state wasn't asked anywhere.